File: tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/assets/TestClasses/AttributesTest.vb
' Licensed to the .NET Foundation under one Or more agreements.
' The .NET Foundation licenses this file to you under the MIT license.

Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace TestClasses
    <AllowHtml()> ' Comment
    Public Class Class1
    End Class

    <System.AllowHtml()>
    <OutputCacheAttribute()> ' Comment2
    Public Class Class2
    End Class

    <Web.Mvc.ValidateInput(), System.Web.Mvc.AuthorizeAttribute>
    Public Class Class3
    End Class

    <Microsoft.AspNetCore.Mvc.Filters.ActionFilter()>
    <System.Web.Mvc.Authorize()>
    <ValidateInputAttribute()>
    Public Class Class4
        <Bind()>
        <Microsoft.AspNetCore.Authorization.AuthorizeAttribute>
        Public Sub Method1(
        <BindAttribute> ByVal s As String)
        End Sub
    End Class
End Namespace

File: tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/assets/TestClasses/AttributesTest.cs
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test.assets.TestClasses
{
    [AllowHtml] //
    public class Class1
    {
    }

    [System.AllowHtml] //
    [OutputCacheAttribute]
    public class Class2
    {
    }

    [Web.Mvc.ValidateInput,System.Web.Mvc.AuthorizeAttribute]
    public class Class3
    {
    }

    [Microsoft.AspNetCore.Mvc.Filters.ActionFilter]
    [System.Web.Mvc.Authorize]
    [ValidateInputAttribute]
    public class Class4
    {
        [Bind]
        [Microsoft.AspNetCore.Authorization.Authorize]
        public void Method1([BindAttribute] string s)
        {

        }
    }
}
